docs/configuration.md

```diff
@@ -18,10 +18,31 @@ The following example configuration assumes that:
 
 ### Channels
 
-In order to be able to connect to your node as a channel participant you need to have
-a port set up for channels' websocket communication. That would be the TCP port (`websocket` > `channel` > `port` parameter).
-
-Please notice that, if your node is behind a firewall, you need to open a TCP port in your firewall (the one you want to connect to) and map that port to the one the node actually listens on (`websocket` > `channel` > `port` parameter).
+`epoch` provides an infrastructure for using state channes. There are two
+distinct protocols involved:
+* WebSocket client one
+* Noise encoded one
+
+#### Channels' WebSocket client setup
+
+In order to connect as a WebSocket client, one must set up a port and a host
+the service is to listen at. This is a private node setting that is left for
+the node operator to secure its access. The TCP port can be set using
```

Comment:
> * Which of the two items is this sentence referring to?

So because of the heading of the section this is referring to ["External Channels WebSocket endpoint"](https://github.com/aeternity/protocol/blame/dde9afac2709fd02f3a770aff6f98558bc011c63/epoch/api/README.md#L31-L34) that is described as:

> * It is defined in the rest of this document;
> * It is meant to be exposed on the Internet;
> * Its TCP port is configurable.

Reply:
So narrowing down this question:

This text says basically that the endpoint is not meant to be exposed on the Internet ("is left for
the node operator to secure its access") while the the API overview in the protocol repository says the [opposite](https://github.com/aeternity/protocol/blame/dde9afac2709fd02f3a770aff6f98558bc011c63/epoch/api/README.md#L33). Is the endpoint meant to be exposed on the Internet?

Reply:
> meant to be exposed on the Internet?

I mean whether it makes sense in order to fulfill any use cases, and whether it is safe.

Reply:
As discussed, [this](https://github.com/aeternity/protocol/blame/dde9afac2709fd02f3a770aff6f98558bc011c63/epoch/api/README.md#L31) shall be internal not external.

Reply:
Addressed in aeternity/protocol#244